tests/ui/panics/short-ice-remove-middle-frames.rs

```diff
@@ -0,0 +1,57 @@
+// compile-flags:-Cstrip=none
+// run-fail
+// check-run-results
+// exec-env:RUST_BACKTRACE=1
+// ignore-android FIXME #17520
+// ignore-wasm no panic support
+// ignore-openbsd no support for libbacktrace without filename
+// ignore-emscripten no panic or subprocess support
+// ignore-sgx no subprocess support
```

Comment:
I'm a bit confused, where is this test using a subprocess? It's ok to ignore the SGX platform as backtraces are currently not symbolized

Reply:
This part is copied from another test case which also involves short backtraces `tests/ui/panics/issue-47429-short-backtraces.rs`, I will update the note part.